api/client/constants.py

```diff
@@ -0,0 +1,37 @@
+"""Constants about the Gro ontology that can be imported and re-used anywhere."""
+
+REGION_LEVELS = {
```

Comment:
minor. same constant in lib, referred to in client

Reply:
Whoops. Stuff got added back in when resolving a merge conflict. Removed. Now it's importing from the constants file